crates/proto/go/Cargo.toml

```diff
@@ -6,6 +6,7 @@ edition = "2021"
 [dependencies]
 proto_core = { path = "../core" }
 log = "0.4.17"
+moon_utils = { path = "../../core/utils" }
```

Comment:
proto cant import the moon stuff, since itll be pulled out into another repo

Reply:
ooo right